e2e-with-targets/01_creating_database.R

```diff
@@ -0,0 +1,38 @@
+## database.R
+
+# loading libraries -------------------------------------------------------------------------------
+box::use(
+  dplyr[mutate, relocate, where, select],
+  RSQLite[SQLite],
+  DBI[dbConnect, dbWriteTable, dbDisconnect]
+)
+
+# preparing data sets for database ----------------------------------------------------------------
+mtcars <- datasets::mtcars
+iris <- datasets::iris
+
+mtcars_df_proper <- mtcars |>
+  mutate(name = rownames(mtcars)) |>
+  `rownames<-`(NULL) |>
```

Comment:
Maybe we can use tibble functions?
```suggestion
  tibble::rownames_to_column("name") |>
```